api/tests/integration/scripts/certification/next-gen/import-complementary-alone-feature-pilot-certification-centers-from-csv_test.js

```diff
@@ -0,0 +1,151 @@
+import { main } from '../../../../../scripts/certification/next-gen/import-complementary-alone-feature-pilot-certification-centers-from-csv.js';
+import { CERTIFICATION_FEATURES } from '../../../../../src/certification/shared/domain/constants.js';
+import { catchErr, createTempFile, databaseBuilder, expect, knex, removeTempFile } from '../../../../test-helper.js';
+
+describe('Integration | Scripts | Certification | import-pilot-certification-centers-from-csv', function () {
+  let file;
+
+  afterEach(async function () {
+    await removeTempFile(file);
+  });
+
+  context('when pilot certification center list from a csv file is imported', function () {
+    it('should import the pilot certification center list', async function () {
+      // given
+      const file = 'pilot-certification-centers-valid.csv';
+      const data = 'certification_center_id;\n1001;\n1002;\n';
+      const csvFilePath = await createTempFile(file, data);
+
+      const certificationCenterId1 = 1001;
+      const certificationCenterId2 = 1002;
+
+      const featureId = databaseBuilder.factory.buildFeature({
+        key: CERTIFICATION_FEATURES.CAN_REGISTER_FOR_A_COMPLEMENTARY_CERTIFICATION_ALONE.key,
+      }).id;
+
+      databaseBuilder.factory.buildCertificationCenter({ id: certificationCenterId1, isV3Pilot: true });
+      databaseBuilder.factory.buildCertificationCenter({ id: certificationCenterId2, isV3Pilot: true });
+
+      await databaseBuilder.commit();
+
+      // when
+      await main(csvFilePath);
+
+      // then
+      const pilotCertificationCenterList = await knex('certification-center-features').select(
+        'certificationCenterId',
+        'featureId',
+      );
+
+      expect(pilotCertificationCenterList).to.deep.equal([
+        { certificationCenterId: certificationCenterId1, featureId },
+        { certificationCenterId: certificationCenterId2, featureId },
+      ]);
+    });
+
+    context('when pilot certification centers are already imported', function () {
+      it('should removes pilot certification centers for the complementary alone feature from the list', async function () {
```

Comment:
Peut-on raccourcir ainsi ?
  ```suggestion
      it('should removes pilot certification centers from the list', async function () {
```

Reply:
'should removes existing pilot certification centers from the list' ?

Reply:
J'ai tout de même besoin de spécifier qu'ils sont pilotes pour la séparation pix/pix+, car le tests suivant spécifie de ne pas supprimer les centres pilotes pour une autre feature.

Dans la proposition on perd cette info ...

Reply:
Wording modifié 👍